hedera-node/src/main/java/com/hedera/services/fees/TxnRateFeeMultiplierSource.java

```diff
@@ -160,7 +160,11 @@ public void resetCongestionLevelStarts(Instant[] savedStartTimes) {
 
 	@Override
 	public Instant[] congestionLevelStarts() {
-		return congestionLevelStarts;
+		/* If the Platform is serializing a fast-copy of the MerkleNetworkContext,
+		and that copy references this object's congestionLevelStarts, we will get
+		a (transient) ISS if the congestion level changes mid-serialization on one
+		node but not others. */
+		return congestionLevelStarts.clone();
```

Comment:
`clone()` is still a shallow copy. Shall a deep copy of the Instant array be used here?

Reply:
Shallow copy is fine b/c `Instant` is immutable...so as long as we have our own **array** object, we know it will never change.